0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -213,6 +214,13 @@ Submit a fulfillment to a ledger. Plugin must be connected, otherwise the promis
 
 **TODO**: Define what the message format is.  Plugin must be connected, otherwise the promise should reject.
 
+#### rejectIncomingTransfer
+<code>ledgerPlugin.rejectIncomingTransfer( **transferId**:String ) ⇒ Promise.&lt;null></code>
```

Comment:
I think this should include a return message (of type `Buffer`), so we can give more context why we rejected the transfer.